scripts/exrrfs_jedivar.sh

```diff
@@ -159,3 +166,6 @@ fi
 cp "${DATA}"/jdiag* "${COMOUT}/jedivar/${WGF}"
 cp "${DATA}"/jedivar*.yaml "${COMOUT}/jedivar/${WGF}"
 cp "${DATA}"/log.out "${COMOUT}/jedivar/${WGF}"
+cp "${DATA}"/out_abi_g16.satbias.nc "${COMOUT}/jedivar/${WGF}/abi_g16.satbias.nc"
+cp "${DATA}"/out_abi_g16.satbias_cov.nc "${COMOUT}/jedivar/${WGF}/abi_g16.satbias_cov.nc"
```

Comment:
Will every satellite generate an out*stabias.nc and an out*satibas_cov.nc file?

Reply:
yes

Reply:
For GETKF, are these `out*stabias.nc` files generated in the observer step or the solver step?  @xyzemc @SamuelDegelia-NOAA

Reply:
There is no out*satbias* from  GETKF either observer or solver since the satbias are not going to update in GETF.